Signal/src/AppDelegate.m

```diff
@@ -172,10 +172,13 @@ - (UIViewController *)loadingRootViewController
     BOOL shouldShowUpgradeLabel = NO;
     NSString *lastAppVersion = AppVersion.instance.lastAppVersion;
     NSString *lastCompletedLaunchAppVersion = AppVersion.instance.lastCompletedLaunchAppVersion;
+    // Every time we change a database view in such a way that might cause a delay on launch,
+    // we need to bump this constant.
+    NSString *lastVersionWithDatabaseViewChange = @"2.13.0";
     BOOL mayNeedUpgrade = ([TSAccountManager isRegistered] &&
-        [VersionMigrations isVersion:lastAppVersion atLeast:@"2.0.0" andLessThan:@"2.13.0"]);
+        [VersionMigrations isVersion:lastAppVersion atLeast:@"2.0.0" andLessThan:lastVersionWithDatabaseViewChange]);
     BOOL hasCompletedUpgrade = (lastCompletedLaunchAppVersion &&
-        [VersionMigrations isVersion:lastCompletedLaunchAppVersion atLeast:@"2.13.0"]);
+        [VersionMigrations isVersion:lastCompletedLaunchAppVersion atLeast:lastVersionWithDatabaseViewChange]);
```

Comment:
I just wanted to constant-ize this and add a comment, but we don't need to bump the constant with this PR.